diff --git a/lean/src/interp/sym/defs.lean b/lean/src/interp/sym/defs.lean
index af051eb..6a1be98 100644
--- a/lean/src/interp/sym/defs.lean
+++ b/lean/src/interp/sym/defs.lean
@@ -90,7 +90,7 @@ def interpS : ℕ → lang.exp D O → env SymV → sym.state SymB → option (s
             end
           end          
     end
-  | error := some (halt (f.assert σ f.mk_ff))
+  | error := some (halt (f.assume σ f.mk_ff))
   | abort := some (halt (f.assume σ f.mk_ff))
   end
 
